AvatarChatbot/docker/Dockerfile

```diff
@@ -0,0 +1,34 @@
+
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+FROM python:3.11-slim
+
+RUN apt-get update -y && apt-get install -y --no-install-recommends --fix-missing \
+    libgl1-mesa-glx \
+    libjemalloc-dev \
+    vim \
+    git
+
+RUN useradd -m -s /bin/bash user && \
+    mkdir -p /home/user && \
+    chown -R user /home/user/
+
+WORKDIR /home/user/
+RUN git clone https://github.com/ctao456/GenAIComps.git
+WORKDIR /home/user/GenAIComps
+RUN git checkout ctao/animation
```

Comment:
same here. suggest not to use personal branch
```suggestion
```

Reply:
Fixed